bin/lib/getStorybookInfo.js

```diff
@@ -122,7 +122,7 @@ const findViewlayer = async ({ env, log, options, packageJson }) => {
       resolvePackageJson(pkg)
         .then((json) => ({ viewLayer, version: json.version }))
         .catch(() => Promise.reject(new Error(noViewLayerPackage(pkg)))),
-      timeout(10000),
+      timeout(9000),
```

Comment:
Lol what's up with that one second? Fine with me though.

Reply:
10000 is exactly the CI timeout lol, I lowered it, but then found I should really just force the timers to end, will revert these changes.